stories/Atom/Images/ImageCaptionCredit/ImageCaptionCredit.jsx

```diff
@@ -14,22 +19,17 @@ export const Images = ({
 
   return (
     <figure data-viewport="true" className={cls('image-figcaption', `${size}`)}>
-      <div data-viewport="true" className='image-figcaption__cart scale-up'>
+      <div data-viewport="true" className="image-figcaption__cart scale-up">
```

Comment:
Can you refactor this so we avoid excessive markup? I believe required treatment can be encapsulated in css, without a need for additional <div> and classes. Also, please make sure default "wide" treatment is applied out of the box on `<figure><img ...></figure>` without any additional classes